core/tst/software/aws/toolkits/core/credentials/Mocks.kt

```diff
@@ -8,7 +8,7 @@ import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider
 import software.aws.toolkits.core.utils.test.aString
 
 fun aToolkitCredentialsProvider(
-    identifier: ToolkitCredentialsIdentifier = aToolkitCredentialsIdentifier(),
+    identifier: CredentialIdentifier = aToolkitCredentialsIdentifier(),
```

Comment:
the `aToolkitCredentialsIdentifier` generator should be renamed to `aCredentialIdentifier` also.

Reply:
Changed